store/store.go

```diff
@@ -609,15 +609,74 @@ func (s *SnapUbuntuStoreRepository) Snap(name, channel string, devmode bool, use
 // Find finds  (installable) snaps from the store, matching the
 // given search term.
 func (s *SnapUbuntuStoreRepository) Find(searchTerm string, channel string, user *auth.UserState) ([]*snap.Info, error) {
-	if channel == "" {
-		channel = "stable"
+	// searchTerm is [<prefix>:]term
```

Comment:
I'd move this entirely to the documentation file above and just reference it here to ensure that this is only defined once and is easy to find in the docs/ directory